### PR TITLE
DataQuery: added table paginator

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

File: webapp/components/DataQuery/Visualizer/Visualizer.js [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
-          offset,
-        })
-      )}
+      {React.createElement(components[Query.getDisplayType(query)], {
+        query,
+        data,
+        dataEmpty,
+        nodeDefsSelectorVisible,
+        offset,
+      })}
```
Comment (ramirobg94) [inline webapp/components/DataQuery/Visualizer/Visualizer.js:23]: are these values going to be the same in each kind of component?

Comment (minotogna) [reply webapp/components/DataQuery/Visualizer/Visualizer.js:23]: at the moment we have only one (Table) and that's what it needs.
But I guess yes, when we'll add other visualization components they won't need more than that

File: webapp/components/DataQuery/store/actions/useFetchCount.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  return {
+    fetchCount: useCallback(
+      ({ query }) =>
+        post({
+          url: `${getUrl({ surveyId, query })}/count`,
+          body: getBody({ cycle, query }),
+        }),
+      [cycle, surveyId, post]
+    ),
+    resetCount: useCallback(() => reset(), [reset]),
```
Comment (SteRiccio) [inline webapp/components/DataQuery/store/actions/useFetchCount.js:28]: can it be just `resetCount: useCallback(reset, [reset])`?

Comment (minotogna) [reply webapp/components/DataQuery/store/actions/useFetchCount.js:28]: it can. will update in a next branch

**Pull request merged**